src/parameters.f90

```diff
@@ -566,6 +640,15 @@ subroutine parameter(input_i3d)
      if (itype==itype_gravitycur) then
         write(*,*)  "Initial front location: ", pfront
      endif
+     ! Check output parameters are valid for TTBL
+     if (itype.eq.itype_ptbl) then
+        if (ioutput < ilist .or. mod(ioutput, ilist) /= 0) then
+           if (nrank == 0) write (*, *) 'ioutput must be exactly divisible by ilist'
```

Comment:
Calling `decomp_2d_abort` is probably better

Reply:
okay, will do that

Reply:
Calling the subroutine `decomp_2d_abort` takes one line and it prints more stuff :
```
call decomp_2d_abort(__FILE__, __LINE__, ioutput, "ioutput must be exactly divisible by ilist")
```
the master rank will print the name of the file containing the error, the line where the error is located, the value of `ioutput` and the error message to both stdout and stderr.

It is shorter to use decomp_2d_abort and it provides more information
https://github.com/2decomp-fft/2decomp-fft/blob/dev/src/decomp_2d_mpi.f90#L154

Reply:
fixed